app/models/inning.rb

```diff
@@ -13,6 +14,10 @@ def self.order_by_name
     order(name: :desc)
   end
 
+  def self.current_and_future
+    where(["current = ? or start_date >= ?", true, Date.today]).order(:start_date)
```

Comment:
Looks great! I think you're making really good use of ActiveRecord to build this query for you as well as Rails' built in date/time helpers